accession-commons-jpa/src/main/java/uk/ac/ebi/ampt2d/commons/accession/persistence/jpa/accession/entities/InactiveAccessionEntity.java

```diff
@@ -18,6 +18,7 @@
 package uk.ac.ebi.ampt2d.commons.accession.persistence.jpa.accession.entities;
 
```

Comment:
can this be uk.ac.ebi.ampt2d.commons.accession.persistence.jpa.entities ??

Reply:
It is possible, @cyenyxe your take?

Reply:
Agree